src/components/ErrorForm/ErrorFormWithList.jsx

```diff
@@ -1,62 +1,49 @@
-import React from 'react'
+import React, { useRef, useState } from 'react'
 import formStyle from './errorForm.module.css'
 
-class ErrorFormWithList extends React.Component {
-  emailRef = React.createRef()
+export default function ErrorFormWithList() {
+  const emailRef = useRef(null)
+  const [emailInputValid, setEmailInputValid] = useState(false)
```

Comment:
At the moment there's a wee issue with the form in that it starts in an invalid state, because we're setting it to false here (if you have a look at the netlify 'deploy preview' details link and go to the 'handling form errors' article you should see it

I think if you make this undefined by default it will solve this issue (as the invalid state is only shown if we're definitely false)